### DISCUSSION TITLE
Select Component: Option to use OS native style picker?

### DISCUSSION THREAD

**Discussion by ongnicolas** [GeekyAnts/NativeBase#5164] [Feature request]
### Description

Hi, I noticed in Android the picker's style is for iOS. Is there a way to use the native android picker??

### Problem Statement

Android shows iOS style picker

![Screenshot_1656436489](https://user-images.githubusercontent.com/66074385/176244090-025bf04f-dc72-48a4-8d2b-c1326d2b91dc.png)

### Proposed Solution or API

option for OS native style picker

**Comment by Krithikj20** [comment]
Hi @madmeatballs, Thank you for reporting it. We will look into this issue.

**Comment by ankit-tailor** [comment]
Hey @madmeatballs, Currently Select is using `ActionSheet` on IOS and android. We'll consider adding this in future release.

**Reply by ongnicolas** [reply]
It would be great to have and option to use native style picker! I hope it'll make it.
